lib/puppet/rest/ssl_context.rb

```diff
@@ -0,0 +1,18 @@
+module Puppet::Rest
+  class SSLContext
+    def self.verify_peer(cert_store)
```

Comment:
I'm not convinced having these constructors makes the calling code read better than just passing in the verify mode to `new`

Reply:
Okay, I'll remove them.